decoder_test.go

```diff
@@ -292,18 +276,12 @@ func TestReadBytes(t *testing.T) {
 		}
 
 		bts, err := d.readBytes(tt.readLen)
-		if err != nil {
-			t.Errorf("got error '%v'", err)
-		}
+		require.NoError(t, err)
 
-		if !reflect.DeepEqual(bts, tt.bytes) {
-			t.Errorf("invalid bytes '%v' '%v'", bts, tt.bytes)
-		}
+		assert.Truef(t, reflect.DeepEqual(bts, tt.bytes), "invalid bytes '%v' '%v'", bts, tt.bytes)
```

Comment:
Please consider using assert.Equal and put expected and actual arguments in a correct order.